src/core/Object3D.js

```diff
@@ -719,6 +719,26 @@ class Object3D extends EventDispatcher {
 
 		}
 
+		if ( this.isBatchedMesh ) {
+
+			object.type = 'BatchedMesh';
+			object._drawRanges = this._drawRanges;
+			object._reservedRanges = this._reservedRanges;
+
+			object._visible = this._visible;
+			object._active = this._active;
+
+			object._maxGeometryCount = this._maxGeometryCount;
+			object._maxVertexCount = this._maxVertexCount;
+			object._maxIndexCount = this._maxIndexCount;
+
+			object._geometryInitialized = this._geometryInitialized;
+			object._geometryCount = this._geometryCount;
+
+			object._matricesTexture = this._matricesTexture.toJSON();
```

Comment:
Can we avoid serializing the internal texture? Ideally, it is just recreated in `BatchedMesh` if it is missing.

Besides, can we avoid the underscore syntax in the JSON object? 

Also, at some point this logic should be moved to `BatchedMesh.toJSON()`. `Object3D` should access private variables of other subclasses.

Reply:
> Can we avoid serializing the internal texture? Ideally, it is just recreated in BatchedMesh if it is missing.

This texture is how the matrices are canonically stored. I can generate a list of serialized matrices from the texture instead, though, if that's acceptable.

> Besides, can we avoid the underscore syntax in the JSON object?

Yup! I'll fix it.

> Also, at some point this logic should be moved to BatchedMesh.toJSON(). Object3D should not access private variables of subclasses.

I thought so, too, but this looked like how all the other classes like InstancedMesh were working. Though it's unclear to me why this is needed.

Reply:
> This texture is how the matrices are canonically stored. I can generate a list of serialized matrices from the texture instead, though, if that's acceptable.

I forgot after #27130 there is only the texture that holds the matrices. In this case, I'm fine with serializing directly. the texture

Reply:
BTW: `ObjectLoader` should also be able to handle `BatchedMesh`. Maybe we can add this to the todo list.